src/main/java/com/mercuryCyclists/procurement/controller/SupplierController.java

```diff
@@ -44,6 +45,9 @@ public class SupplierController {
     @PutMapping()
     public Supplier updateSupplier(@RequestBody Supplier supplier) { return supplierService.updateSupplier(supplier); }
 
+    @PutMapping("/{supplierId}")
```

Comment:
We need to decide if we want to comment on each method or not. I find it generally standard, but our code is so simplistic

Reply:
Also, following REST standards this is pretty sketch.

How would the user know the difference between the URLs?
A better approach might be with a contact controller
POST or PUT /api/v1/contact/{supplierID}
Then the client knows exactly what they are doing

Reply:
If you have time could we have a talk about this, as I think that following the standard you are correct however I did it that way since contact needs to be connected to a supplier